src/window_draw_list.rs

```diff
@@ -209,6 +209,19 @@ impl<'ui> WindowDrawList<'ui> {
         Circle::new(self, center, radius, color)
     }
 
+    /// Draw a text whose upper-left corner is at point `pos`.
+    pub fn add_text<P, C>(&self, pos: P, col: C, text: &str)
+    where
+        P: Into<ImVec2>,
+        C: Into<ImColor>,
+    {
+        unsafe {
+            let start = text.as_ptr() as *const i8;
```

Comment:
Please use `*const c_char`. `c_char` is not always `i8` (on ARM for example it's `u8` so this won't compile)

Reply:
Thx! Pushed the fix on 9b0804f